src/api/supabase/queries/cases.ts

```diff
@@ -2,13 +2,20 @@ import { UUID } from 'crypto';
 import supabase from '../createClient';
 
 export async function getAllCases() {
-  const { data, error } = await supabase.from('cases').select();
+  const { data, error } = await supabase
+    .from('cases')
+    .select()
+    .or('needs_interpreter.eq.true,needs_attorney.eq.true');
```

Comment:
we might need to update these after the switch to using the database function for the queries. May or may not also need to update the database func?

Reply:
yeah we should update the database function to check that as well! I can update the code here so hopefully there's no merge conflicts